contracts/modules/cover/Cover.sol

```diff
@@ -91,7 +92,10 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
   // Eg. coverAssetsFallback = 3 (in binary 11) means assets at index 0 and 1 are supported.
   uint32 public coverAssetsFallback;
 
-  // TODO: implement using buckets
+  // The last time activeCoverExpirationBuckets was updated
+  uint public lastBucketUpdateId;
```

Comment:
We need to track the bucket ids for each asset separately, otherwise you'll only expire cover amounts on one asset and skip updates on others. This can be packed nicely together with `totalActiveCoverInAsset` that you have below. Creating a struct and limiting its size to a single slot should do it.